.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "tabWidth": 4,
+  "overrides": [
+    {
+      "files": ["*.json", ".prettierrc"],
+      "options": {
+        "tabWidth": 2
+      }
+    }
+  ]
+}
```

Comment:
Please try to comply with New Mendix pattern:
```
{
  "trailingComma": "none",
  "useTabs": false,
  "tabWidth": 4,
  "semi": true,
  "singleQuote": false,
  "printWidth": 120,
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "arrowParens": "avoid",
  "proseWrap": "always"
}
```

Reply:
Done, but what about the overrides for json files? Andries told me json files should have tabwidth 2.